SpatialGDK/Source/SpatialGDK/Public/Interop/CrossServerRPCParams.h

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Connection/SpatialGDKSpanId.h"
+#include "Schema/RPCPayload.h"
+#include "SpatialCommonTypes.h"
+
+namespace SpatialGDK
+{
+struct FCrossServerRPCParams
+{
+	FCrossServerRPCParams(const FUnrealObjectRef& InObjectRef, const Worker_RequestId_Key InRequestId, RPCPayload&& InPayload,
+						  const uint32 InTimeoutMillis, const FSpatialGDKSpanId& InSpanId)
+		: ObjectRef(InObjectRef)
+		, Payload(MoveTemp(InPayload))
+		, RequestId(InRequestId)
+		, Timestamp(FDateTime::Now())
+		, TimeoutMillis(InTimeoutMillis)
+		, SpanId(InSpanId)
+	{
+	}
+
+	// Moveable, not copyable.
+	FCrossServerRPCParams() = delete;
+	FCrossServerRPCParams(const FCrossServerRPCParams&) = delete;
+	FCrossServerRPCParams(FCrossServerRPCParams&&) = default;
+	FCrossServerRPCParams& operator=(const FCrossServerRPCParams&) = delete;
+	FCrossServerRPCParams& operator=(FCrossServerRPCParams&&) = delete;
```

Comment:
Non nit - Should this one be `default`?

Nit - I'd put the deleted default constructor and the destrcutor in a block above this one.